src/memory/transfer_context.h

```diff
@@ -46,13 +46,10 @@ class TransferContext : public JavaInstanceWrapper {
         for (uint32_t i = 0; i < args_size; ++i) {
             ktvariant::get_variant_from_buffer(buffer, p_args[i]);
         }
-
-        buffer->rewind();
     }
 
     _FORCE_INLINE_ static void write_return_value(SharedBuffer* buffer, const Variant& r_ret) {
         ktvariant::send_variant_to_buffer(r_ret, buffer);
-        buffer->rewind();
```

Comment:
Why this method does not rewind ?
This would avoid to rewind before calling it in icall method.

Reply:
It's simply to keep all the rewind() calls in a single files, so it's easier to follow. I'd rather just erasing this function than putting rewind back in it.

Reply:
I'm ok with deleting it. This way all is in cpp and is still consistent